packages/herodotos/herodotos.1.0/opam

```diff
@@ -0,0 +1,44 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Tracking code positions through software versions"
+description: """
+Herodotos is open source and can be freely redistributed under the terms of the GNU General Public
+License version 2. See the file LICENSE in the distribution for licensing information."""
+maintainer: ["Nicolas Palix"]
+authors: ["Nicolas Palix" "Oscar Granger" "Justin Nguyen" "Vincent Arnone"]
+license: "GPL-2.0-or-later"
+tags: ["coccinelle" "ocaml" "menhir" "volt" "parmap" "postgresql" "dune"]
+homepage: "https://github.com/coccinelle/herodotos"
+doc: "https://coccinelle.gitlabpages.inria.fr/website/herodotos/herodotos.html"
+bug-reports: "https://github.com/coccinelle/herodotos/issues"
+depends: [
+  "ocaml"
+  "dune" {>= "3.1"}
+  "odoc" {with-doc}
+  "vlt" {>= "0.1"}
+  "parmap" {>= "1.2.5"}
+  "postgresql" {>= "5.0.0"}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
```

Comment:
```suggestion
    "@install"
```
I think this will not do anything. The dune file does not provide a `public_name` and indeed it seems that the installation is done manually via a Makefile on the repo

See:
- https://github.com/coccinelle/herodotos/blob/main/herodotos/dune#L1
- https://github.com/coccinelle/herodotos/blob/main/herodotos/Makefile#L166

Reply:
This needs to be fixed before the release can be merged. The easiest solution is adding
```diff
(executable
+       (public_name herodotos)
	(name herodotos)
```